plugins/scaffolder-actions/scaffolder-backend-module-http-request/README.md

```diff
@@ -27,9 +27,9 @@ Configure the action:
 
 ```typescript
 // packages/backend/src/plugins/scaffolder.ts
-
+const discovery = UrlPatternDiscovery.compile(config.getOptionalString("backend.baseUrl") + "/{{pluginId}}") 
```

Comment:
This should already be passed in via the PluginEnvironment. There would be no need to construct it here.

See here for example in the backstage example backend.

https://github.com/backstage/backstage/blob/master/packages/backend/src/types.ts#L36

Reply:
Updated.